nservicebus/gateway/index.md

```diff
@@ -8,49 +8,46 @@ related:
 
-The gateway doesn't support pub/sub (more on that later) but this isn't a problem since request/response is perfectly fine within a BS, remembering that those sites are physically different but the communication is within the same logical BS. So when you use the gateway, the guideline is to model the messages going explicitly across sites. If you extend the sample to include a sales service responsible for reporting the sales statistics so that the pricing service can set appropriate prices, you get the following picture:
+The gateway doesn't support pub/sub (more on that later) but this isn't a problem since request/response is perfectly fine within a BS, remembering that those sites are physically different but the communication is within the same logical BS. So when using the gateway, the guideline is to model the messages going explicitly across sites. The following picture illustrates the sample and includes a sales service responsible for reporting the sales statistics so that the pricing service can set appropriate prices.
 
 ![Gateway Store and Headquarters example](store-to-headquarters-pricing-and-sales.png "Logical view")
 
 The prices are pushed daily to the stores and sales reports are pushed daily to the HQ. Any pub/sub goes on within the same physical site. This is the reason that the NServiceBus gateway doesn't support pub/sub across sites since it shouldn't be needed in a well designed system.
 
-Going across sites usually means radically different transport characteristics like latency, bandwidth, reliability, and explicit messages for the gateway communication, helping to make it obvious for developers that they are about to make cross-site calls. This is where Remote Procedure Call (RPC) really starts to break down.
-
-RPC completely hides the fact that you are now going out of the data center and will meet all the fallacies of distributed computing head on.
-
+Going across sites usually means radically different transport characteristics like latency, bandwidth, reliability, and explicit messages for the gateway communication, helping to make it obvious for developers that they are about to make cross-site calls. This is where Remote Procedure Call (RPC) really starts to break down as it will meet all the fallacies of distributed computing head on.
 
 ## Using the gateway
 
 In order to send message to other sites call `SendToSites` method:
 
 snippet:SendToSites
 
-This allows you to pass in a list of sites to where you want to send the messages. You can configure each site with a different transport mechanism. Currently the supported channels are HTTP/HTTPS but you can easily extend the gateway with a custom implementation.
+`SendToSite` accepts a list of sites to send the messages to. Each site can be configured with a different transport mechanism. Currently the supported channels are HTTP/HTTPS but the gateway can be extended with a custom implementations.
```

Comment:
There are no samples for this. I'll add a section about what needs to be done to provide a custom implementation.